pkg/controller/installation/installation_controller.go

```diff
@@ -598,11 +601,11 @@ func (r *ReconcileInstallation) handleUninstall(installation *integreatlyv1alpha
 			return retryRequeue, merr
 		}
 
-		logrus.Infof("uninstall completed")
+		log.Info("uninstall completed")
```

Comment:
Hey Brian, just a question more than anything else, is there a reason why we sometimes log an info message starting with capital letters and sometimes we don't?

Reply:
We don't have a standard for that